hr_timesheet_employee_analytic_tag/models/account_analytic_line.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2022 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class AccountAnalyticLine(models.Model):
+    _inherit = "account.analytic.line"
+
+    @api.model_create_multi
+    def create(self, vals_list):
+        employee_model = self.env["hr.employee"]
+        for vals in vals_list:
+            if vals.get("employee_id"):
```

Comment:
You should test as well if there's a project for saying that is a timesheet:
```suggestion
            if vals.get("employee_id") and vals.get("project_id"):
```